tutorminio/patches/local-docker-compose-services

```diff
@@ -4,13 +4,21 @@ minio:
   {% if not MINIO_GATEWAY %}
   volumes:
     - ../../data/minio:/data
+  {% elif MINIO_GATEWAY == "gcs" %}
+  volumes:
+    - ../plugins/minio/apps/minio_secrets:/minio_secrets
   {% endif %}
   environment:
     MINIO_ROOT_USER: "{{ OPENEDX_AWS_ACCESS_KEY }}"
     MINIO_ROOT_PASSWORD: "{{ OPENEDX_AWS_SECRET_ACCESS_KEY }}"
-  {% if MINIO_GATEWAY %}
-  command: gateway {{ MINIO_GATEWAY }} --address ":9000" --console-address ":9001"
-  {% else %}
+    {% if MINIO_GATEWAY == "gcs" %}
+    GOOGLE_APPLICATION_CREDENTIALS: /minio_secrets/google_application_credentials.json
```

Comment:
Note: I believe we need to have a service account with Storage Object Viewer / Creator permissions

Reply:
yes, I forgot to mention that 😁

Reply:
I created our generic service account, with Storage Object User. According to documentation it has viewer+writting among others that might be useful for us.

> Grants access to create, view, list, update, and delete objects and managed folders, along with their metadata. Does not give permission to get or set ACLs or IAM policies.